guidelines/2016-03-08-github-guidelines.md

```diff
@@ -0,0 +1,116 @@
+# GitHub Guidelines
+
+`Editor note: This is the first draft of the GitHub guidelines for Origami.`
+
+
+## How to contribute to Origami
+
+Firstly, thank you for deciding to contribute to Origami! Please find the guidelines for creating issues or pull requests listed below.
+
+
+### Issues
+
+The first steps before creating an issue of any type on the repository is to search through the repository's Open/Closed issues to see whether there's a ticket that replicates the same issue you are having and if there is a fix in progress or has been fixed (alternatively you can use GitHub's search feature for this). If there is no same issue that you are experiencing, then open a new ticket on the affected repository.
+
+#### Reporting bugs
+
+Found a bug with Origami? Fill out the template below before submitting an issue. Due to a recent new addition to GitHub, the template can be included automatically when an issue is opened.
```

Comment:
Hows about we add a little "why" here too. So:

"Found a bug with Origami? **It will help us to prioritise and fix the bug if you **fill out the template below before submitting an issue. Due to a recent new addition to GitHub, the template can be included automatically when an issue is opened.